packages/bpk-component-barchart/README.md

```diff
@@ -58,6 +58,7 @@ export default () => (
 | yAxisMargin                             | number                                | false    | 2.625            |
 | yAxisTickValue                          | func                                  | false    | identity         |
 | yAxisNumTicks                           | number                                | false    | null             |
+| yAxisDomain                             | array                                 | false    | See prop details |
```

Comment:
I think it would be clearer if we call out that the component will calculate these if not provided

```suggestion
| yAxisDomain                             | array                                 | false    | calculated by component |
```

Reply:
Fixed.  Had to update the padding for the whole table.